src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs

```diff
@@ -20,7 +20,7 @@ internal class PickMembersDialogViewModel : AbstractNotifyPropertyChanged
             IGlyphService glyphService,
             ImmutableArray<ISymbol> members)
         {
-            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).OrderBy(s => s.MemberName).ToList();
+            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).ToList();
```

Comment:
It was.  I don't want the items sorted by name in the final UI.  I want them ordered as we found them in the inheritance hierarchy.